server/controllers/v2Manage/outOfServiceBedsController.test.ts

```diff
@@ -315,14 +334,119 @@ describe('OutOfServiceBedsController', () => {
     })
 
     it('redirects to the current temporality if a stray temporal URL parameter is entered', async () => {
-      const indexRequest = { ...request, params: { temporality: '123' } }
+      const apAreaId = 'abc'
+      const indexRequest = { ...request, params: { temporality: '123' }, query: { premisesId, apAreaId } }
 
       const requestHandler = outOfServiceBedController.index()
 
       await requestHandler(indexRequest, response, next)
 
       expect(response.redirect).toHaveBeenCalledWith(paths.v2Manage.outOfServiceBeds.index({ temporality: 'current' }))
     })
+
+    it('if value of Ap Areas and Premises is all empty string is passed to the api for both', async () => {
+      const temporality = 'current'
+      premisesId = 'all'
+      const apAreaId = 'all'
+      const paginatedResponse = paginatedResponseFactory.build({
+        data: outOfServiceBedFactory.buildList(1),
+      }) as PaginatedResponse<OutOfServiceBed>
+      const paginationDetails = {
+        hrefPrefix: `${paths.v2Manage.outOfServiceBeds.index.pattern}?${createQueryString({
+          temporality,
+          premisesId,
+        })}`,
+        pageNumber: 1,
+        sortBy: 'roomName',
+        sortDirection: 'desc',
+      }
+
+      outOfServiceBedService.getAllOutOfServiceBeds.mockResolvedValue(paginatedResponse)
+      ;(getPaginationDetails as jest.Mock).mockReturnValue(paginationDetails)
+
+      const indexRequest = { ...request, params: { temporality }, query: { apAreaId, premisesId } }
+
+      const requestHandler = outOfServiceBedController.index()
+
+      await requestHandler(indexRequest, response, next)
+
+      expect(outOfServiceBedService.getAllOutOfServiceBeds).toHaveBeenCalledWith({
```

Comment:
Given the `it` statement for this test here I think we should do `toHaveBeenCalledWith(expect.objectContaining({ apAreaId: '', premisesId: ''}))`